app/routes/docs.notices.unifiedschema.mdx

```diff
@@ -0,0 +1,191 @@
+---
+handle:
+  breadcrumb: Unified Schema
+---
+
+import {
+  Link,
+  ProcessList,
+  ProcessListHeading,
+  ProcessListItem,
+} from '@trussworks/react-uswds'
+
+import { Highlight } from '~/components/Highlight'
+import { Tab, Tabs } from '~/components/tabs/Tabs'
+
+import { feature } from '~/lib/env.server'
+
+export function loader() {
+  return feature('SCHEMA') || new Response(null, { status: 404 })
+}
+
+# GCN Unified Schema
+
+The GCN Unified Schema provides a standardized method for organizing GCN notices with common properties across different missions following the uniform unit system. It utilizes JSON (JavaScript Object Notation), a lightweight and easily machine-readable data-interchange format, to define the notice schema. The set of core schemas serves as the foundation for creating new GCN Notices alerts, also allows the inclusion of instrument-specific parameters.
+
+## Fork the GitHub Repository
+
+To begin, initiate by forking your own copy of the [nasa-gcn/gcn-schema](https://github.com/nasa-gcn/gcn-schema) repository on GitHub. Next, proceed to clone a local version and follow the project configuration instructions available [instructions](https://gcn.nasa.gov/docs/contributing).
+
+## Setting Up Your Repository
+
+The repository's structure allows producers to create their own folders to define schemas on an instrument-specific level. The schema folder is named according to the format <code>gcn.notices.<i>mission</i></code>, where <i>mission</i> is your [Kafka Topic](/docs/notices/producers). Within it, you can define your own schema and example files. For example, schema definitions for IceCube alerts are located in the directory [`gcn/notices/icecube/`](https://github.com/nasa-gcn/gcn-schema/tree/main/gcn/notices/icecube).
```

Comment:
Please update these instructions to point to the new GitHub primer (/docs/contributing/github).